src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,60 @@
+package edu.kis.vh.nursery;
+
+/**
+ *
+ */
+
+public class DefaultCountingOutRhymer {
+	/**
+	 * @param in
+	 */
+
+	public static final int CAPACITY = 12;
+	public static final int EMPTY = -1;
+	private final int[] NUMBERS = new int[CAPACITY];
+
+	private int total = EMPTY;
```

Comment:
Task 2 in "accessory i hermetyzacja" concerned 'total' getter!
This constant should be different (separate) than the one in peekaboo and countOut methods.

Reply:
You haven't applied this comment. peekaboo and countOut methods shouldn't return EMPTY. They should return another (new) constant. EMPTY is an index of numbers array, and peekaboo and countOut methods return values from this table (or -1 by default). Both -ones are from different domains.

Reply:
You also haven't generated a total getter (getTotal method)